### ISSUE TITLE
rt-view shows a lot of json parsing errors

### ISSUE THREAD

**Issue by Sailoshi** [input-output-hk/cardano-rt-view#156]
My rt-view is getting a lot of parsing errors like the following.
I don't know whether it is a missconfiguration or a general problem with errors produced by the cardano-node.

![image](https://user-images.githubusercontent.com/46825645/108592847-41a81d00-7370-11eb-9968-a48d6e24c2cb.png)

**Comment by denisshevchenko** [comment]
> I don't know whether it is a missconfiguration or a general problem with errors produced by the cardano-node.

@pheelLikeWater Yes, these errors are `cardano-node`'s errors, not RTView's errors. And yes, the reason is an incorrect json configuration file for your node. For example, if you check your node's configuration file in some json lint - is it correct?

**Comment by Sailoshi** [comment]
Hi, yes I've checked my json file several times already. The following is my configuration. Thanks @denisshevchenko.
I have also tried the original one without any changes. Really strange.

`{
  "ApplicationName": "cardano-sl",
  "ApplicationVersion": 1,
  "ByronGenesisFile": "/opt/cardano/cnode/files/byron-genesis.json",
  "ByronGenesisHash": "5f20df933584822601f9e3f8c024eb5eb252fe8cefb24d1317dc3d432e940ebb",
  "LastKnownBlockVersion-Alt": 0,
  "LastKnownBlockVersion-Major": 3,
  "LastKnownBlockVersion-Minor": 0,
  "MaxKnownMajorProtocolVersion": 2,
  "Protocol": "Cardano",
  "RequiresNetworkMagic": "RequiresNoMagic",
  "ShelleyGenesisFile": "/opt/cardano/cnode/files/genesis.json",
  "ShelleyGenesisHash": "1a3be38bcbb7911969283716ad7aa550250226b76a61fc51cc9a9a35d9276d81",
  "TraceBlockFetchClient": true,
  "TraceBlockFetchDecisions": true,
  "TraceBlockFetchProtocol": true,
  "TraceBlockFetchProtocolSerialised": true,
  "TraceBlockFetchServer": true,
  "TraceChainDb": true,
  "TraceChainSyncBlockServer": true,
  "TraceChainSyncClient": true,
  "TraceChainSyncHeaderServer": true,
  "TraceChainSyncProtocol": true,
  "TraceDNSResolver": true,
  "TraceDNSSubscription": true,
  "TraceErrorPolicy": true,
  "TraceForge": true,
  "TraceHandshake": false,
  "TraceIpSubscription": true,
  "TraceLocalChainSyncProtocol": true,
  "TraceLocalErrorPolicy": true,
  "TraceLocalHandshake": false,
  "TraceLocalTxSubmissionProtocol": true,
  "TraceLocalTxSubmissionServer": true,
  "TraceMempool": true,
  "TraceMux": false,
  "TraceTxInbound": false,
  "TraceTxOutbound": false,
  "TraceTxSubmissionProtocol": false,
  "TracingVerbosity": "NormalVerbosity",
  "TurnOnLogMetrics": true,
  "TurnOnLogging": true,
  "MaxConcurrencyDeadline": 2,
  "defaultBackends": [
    "KatipBK"
  ],
  "defaultScribes": [
    [
      "FileSK",
      "/opt/cardano/cnode/logs/node0.json"
    ]
  ],
  "hasEKG": 12788,
  "hasPrometheus": [
    "127.0.0.1",
    12798
  ],
  "minSeverity": "Info",
  "options": {
    "mapBackends": {
      "cardano.node.metrics": [
        "EKGViewBK",
		"TraceForwarderBK"
      ],
      "cardano.node.resources": [
        "EKGViewBK",
		"TraceForwarderBK"
      ]
    },
    "mapSubtrace": {
      "cardano.node.metrics": {
        "subtrace": "Neutral"
      }
    }
  },
  "rotation": {
    "rpKeepFilesNum": 10,
    "rpLogLimitBytes": 50000000,
    "rpMaxAgeHours": 24
  },
  "setupBackends": [
    "KatipBK",
    "EKGViewBK"
  ],
  "setupScribes": [
    {
      "scKind": "FileSK",
      "scName": "/opt/cardano/cnode/logs/node0.json",
      "scFormat": "ScJson",
      "scRotation": null
    }
  ],
  "SocketPath": "/opt/cardano/cnode/sockets/node0.socket",
  "EnableLogging": true,
  "EnableLogMetrics": false,
  "traceForwardTo": {
     "tag": "RemoteSocket",
     "contents": [
       "XXX.XXX.XXX.XXX",
       "XXXX"
     ]
  }
}
`

**Comment by mwolfaardt** [comment]
I have the same, any update?
